### PR TITLE
Add a API documentation library

### PR DESCRIPTION
**Pull request by tochman**
PT chore: https://www.pivotaltracker.com/story/show/114070647
- Adds ApiPipe (apipipe-rails) documentation library
  - https://github.com/Apipie/apipie-rails

capybara-201602211807005289740277

### CODE REVIEW

File: app/controllers/api/v0/ping_controller.rb [hunk 1]
```diff
@@ -1,9 +1,13 @@
 class Api::V0::PingController < ApiController
 
+  api :GET, "/ping/", "Respond with 'Pong'"
+  description "Responds with a message containing the word 'Pong'"
+  formats %w(json text)
+  example " 'message': 'Pnng' "
```
Comment (sampritipanda) [inline app/controllers/api/v0/ping_controller.rb]: Typo.

File: app/controllers/api/v1/users_controller.rb [hunk 2]
```diff
@@ -1,9 +1,19 @@
 class Api::V1::UsersController < ApiController
 
+
+  api :GET, '/v1/users/','List of resources: User'
+  formats ['json']
```
Comment (sampritipanda) [inline app/controllers/api/v1/users_controller.rb]: The routes file says that the formats for this route are `format: /(text|json)/`. This one only has `json`. Which one do we want to stick with? Same with the show action.

File: features/apipipe/apipipe.feature [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Feature: As developer of PU API
+  In order to make usage of my API easier
+  I would like to make my api documentation accessible over internet
+
+  @javascript
+  Scenario: Access API documentation
+    Given I visit the "api-doc" page
+    Then I should see "Project Unify 1.0"
+    And I should see "This describes the resources that make up the official Project Unify public API v1."
+    Then open screenshot
```
Comment (sampritipanda) [inline features/apipipe/apipipe.feature]: Could this be removed?

File: config/routes.rb [hunk 4]
```diff
@@ -1,19 +1,17 @@
 Rails.application.routes.draw do
-
+  apipie
+  resources :users
```
Comment (sampritipanda) [inline config/routes.rb:3]: Where is this resource route used?

Comment (tochman) [inline config/routes.rb:3]: It is used by the administrative interface (Godmin) to access Users

```
https://unify-develop.herokuapp.com/users
```

**Pull request merged**